src/pages/workspaces/workspace/WorkspaceContainer.ts

```diff
@@ -56,121 +66,68 @@ const TitleBarSpinner = (messageComponents) => {
           borderRadius: '0.5rem',
         },
       }),
-      span({ style: { color: colors.dark(), fontSize: 14 } }, messageComponents),
+      span({ style: { color: colors.dark(), fontSize: 14 } }, [props.children]),
     ]),
     style: { backgroundColor: colors.warning(0.1), borderBottom: `1px solid ${colors.warning()}` },
+    onDismiss: () => {},
   });
 };
 
-const AzureWarning = () => {
+const AzureWarning = (): ReactNode => {
   const warningMessage = [
     'Do not store Unclassified Confidential Information in this platform, as it violates US Federal Policy (ie FISMA, FIPS-199, etc) unless explicitly authorized by the dataset manager or governed by your own agreements.',
   ];
-  return TitleBarWarning(warningMessage);
+  return h(TitleBarWarning, warningMessage);
```

Comment:
Nice catch. Using render functions where components were expected has definitely caused confusion in the past.